opentelemetry-api/java-agent/TelemetryFilteredBaseOnSpanEvents/src/main/resources/applicationinsights.json

```diff
@@ -0,0 +1,32 @@
+{
+  "sampling": {
+    "percentage": 100
+  },
+  "preview": {
+    "processors": [
+      {
+        "type": "attribute",
+        "include": {
+          "matchType": "strict",
+          "attributes": [
+            {
+              "key": "myCustomAttributeKey",
+              "value": "myCustomAttributeValue",
+              "type": "string"
+            }
+          ]
+        },
+        "actions": [
+          {
+            "action": "insert",
+            "key": "controlledError",
+            "value": true
+          }
+        ]
+      }
+    ]
```

Comment:
is this telemetry processor needed for the example?

Reply:
yes